src/main/java/smokefree/graphql/CreateInitiativeInput.java

```diff
@@ -4,18 +4,40 @@
 import lombok.AllArgsConstructor;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
+import org.hibernate.validator.constraints.Length;
 import smokefree.domain.Status;
 import smokefree.domain.Type;
 
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
 @Getter
 @NoArgsConstructor(access = AccessLevel.PRIVATE)
 @AllArgsConstructor
 public
 class CreateInitiativeInput {
+
+    @NotNull(message = "The initiativeId must have a value")
```

Comment:
Very nice, but I think actually a bit too elaborate. We have a formal interface, so these errors should never occur. So I don't think we need to specify detailed error message. Also some indication is probably given by the validation framework?